teamwork/apps/projects/views/EditProjectView.py

```diff
@@ -214,7 +91,7 @@ def edit_project(request, slug):
             project.weigh_learn = form.cleaned_data.get('weigh_learn') or 0
             project.project_image = form.cleaned_data.get('project_image')
             project.ta_time = form.cleaned_data.get('ta_time')
-            project.ta_location = form.cleaned_data.get('ta_location')
+            project.ta_location = form.cleaned_data.get('ta_location')            
```

Comment:
Looks like we added some :b: onus whitespice to the end of this line

Reply:
Just fixed.